src/components/Popups/ConfirmationPopup/ConfirmationPopup.tsx

```diff
@@ -1,17 +1,25 @@
 import { FC, ReactNode } from 'react';
-import styles from './ConfirmPopup.module.scss';
 import { useTranslation } from 'react-i18next';
+import styles from './ConfirmPopup.module.scss';
 
 type ConfirmationPopupProps = {
     title: string;
     confirmButtonText: string;
     onCancel: () => void;
-    onSubmit: () => void;
-    children?: ReactNode;
+    onSubmit: () => void | Promise<void>; // может быть и асинхронной // Орлов Дима
```

Comment:
```suggestion
    onSubmit: () => void;
```
Так будет ошибка какая-то? void по идее включает в себя любой тип, в том числе и промис

Reply:
Да. При использовании промиса на этот колбэк линтер ругается